### ISSUE TITLE
Depend on libx11 formulae instead of :x11 XQuartz

### ISSUE THREAD

**Issue by scpeters** [osrf/homebrew-simulation#1227]
Our `ogre*` formulae depend on XQuartz via the `:x11` requirement. The use of `:x11` in homebrew-core has recently been deprecated in favor of the `libx11` homebrew-core formulae (https://github.com/Homebrew/brew/pull/9052). We should consider using these formulae instead of XQuartz. See also https://github.com/Homebrew/homebrew-core/pull/57995, https://github.com/Homebrew/homebrew-core/pull/64165, and https://github.com/Homebrew/homebrew-core/issues/64166.

**Comment by scpeters** [comment]
the `:x11` requirement is now deprecated as of https://github.com/Homebrew/brew/pull/9403, so I'm testing a migration of the ogre* formulae now

**Comment by scpeters** [comment]
the ogre formulae have switched over as of #1241, so I'll close this. please open a new issue if there are problems with the new formulae
